src/pages/Dashboard/index.tsx

```diff
@@ -1,37 +1,160 @@
 /** @jsxImportSource @emotion/react */
+import { Typography } from '@mui/material';
 import BigNumber from 'bignumber.js';
-import React from 'react';
+import { ButtonGroup, Icon, Select, TextField, Toggle, Tooltip } from 'components';
+import React, { InputHTMLAttributes, useContext, useState } from 'react';
+import { useTranslation } from 'translation';
 import { Asset } from 'types';
 
 import { useGetUserMarketInfo } from 'clients/api';
 import { AuthContext } from 'context/AuthContext';
+import { useHideXlDownCss, useShowXlDownCss } from 'hooks/responsive';
 
 import HigherRiskTokensNotice from './HigherRiskTokensNotice';
-import Markets from './Markets';
+import BorrowMarket from './Markets/BorrowMarket';
+import SupplyMarket from './Markets/SupplyMarket';
+import { useStyles } from './styles';
 
 interface DashboardUiProps {
   accountAddress: string;
   userTotalBorrowLimitCents: BigNumber;
+  areHigherRiskTokensDisplayed: boolean;
+  onHigherRiskTokensToggleChange: (newValue: boolean) => void;
+  searchValue: string;
+  onSearchInputChange: (newValue: string) => void;
   assets: Asset[];
 }
 
-const DashboardUi: React.FC<DashboardUiProps> = ({ accountAddress, assets }) => (
-  <>
-    <HigherRiskTokensNotice />
+const DashboardUi: React.FC<DashboardUiProps> = ({
+  accountAddress,
+  assets,
+  areHigherRiskTokensDisplayed,
+  onHigherRiskTokensToggleChange,
+  searchValue,
+  onSearchInputChange,
+}) => {
+  const { t } = useTranslation();
+  const styles = useStyles();
+  const [activeTabIndex, setActiveTabIndex] = useState(0);
 
-    <Markets
-      isXvsEnabled
-      accountAddress={accountAddress}
-      // TODO: refactor to pass just one assets prop
-      supplyMarketAssets={assets}
-      borrowMarketAssets={assets}
-    />
-  </>
-);
+  const showXlDownCss = useShowXlDownCss();
+  const hideXlDownCss = useHideXlDownCss();
+
+  // TODO: add all options
+  const mobileSelectOptions = [
+    {
+      value: 'riskLevel',
+      label: 'Risk level',
```

Comment:
Looks like this should be a translation

Reply:
Yeah, there's a TODO next to the select to add all the potential options. This will come with a next PR